service/src/main/resources/templates/ehr_link_set_template.mustache

```diff
@@ -0,0 +1,49 @@
+<component typeCode="COMP"{{#isNested}} contextConductionInd="true"{{/isNested}}>
+    <LinkSet classCode="OBS" moodCode="EVN">
+        <id root="{{linkSetId}}" />
+        <effectiveTime>
+            {{#effectiveTimeLow}}<low value="{{effectiveTimeLow}}"/>{{/effectiveTimeLow}}{{^effectiveTimeLow}}<low nullFlavor="UNK"/>{{/effectiveTimeLow}}
+            {{#effectiveTimeHigh}}<high value="{{effectiveTimeHigh}}"/>{{/effectiveTimeHigh}}
+        </effectiveTime>
+        <availabilityTime value="{{availabilityTime}}" />
+        {{#conditionNamed}}
+        <conditionNamed typeCode="NAME" inversionInd="true">
+            <namedStatementRef classCode="OBS" moodCode="EVN">
+                <id root="{{conditionNamed}}"/>
+            </namedStatementRef>
+        </conditionNamed>
+        {{/conditionNamed}}
+        {{#relatedClinicalContent}}
+        <component typeCode="COMP">
+            <statementRef classCode="OBS" moodCode="EVN">
+                <id root="{{.}}"/>
+            </statementRef>
+        </component>
+        {{/relatedClinicalContent}}
+        <code code="{{clinicalStatusCode}}" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="{{clinicalStatusDisplay}}">
+            <originalText>{{clinicalStatusDisplay}}, {{qualifierDisplay}}</originalText>
+            <qualifier inverted="false">
+                <name code="{{qualifierCode}}" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="{{qualifierSignificance}}"/>
+            </qualifier>
+        </code>
+    </LinkSet>
+</component>
+{{#generateObservationStatement}}<component>
+    <ObservationStatement classCode="OBS" moodCode="EVN">
+        <id root="{{conditionNamed}}"/>
+        <code><!--code placeholder--></code>
+        <statusCode code="COMPLETE"/>
+        <effectiveTime>
+            {{#effectiveTimeLow}}<low value="{{effectiveTimeLow}}"/>{{/effectiveTimeLow}}{{^effectiveTimeLow}}<low nullFlavor="UNK"/>{{/effectiveTimeLow}}
+            {{#effectiveTimeHigh}}<high value="{{effectiveTimeHigh}}"/>{{/effectiveTimeHigh}}{{^effectiveTimeHigh}}<high nullFlavor="UNK"/>{{/effectiveTimeHigh}}
```

Comment:
Since `<high nullFlavor="UNK"/>` was removed from LinkSet it should be remove here also

> <effectiveTime   ../> | Same as the LinkSet